TUTORIAL.md

```diff
@@ -166,10 +166,8 @@ package main
 
 import (
 	"fmt"
```

Comment:
Style nit.  I prefer the style where imports of the standard library come in one stanza, followed by a blank line, followed by any 3rd party library dependencies, followed by a blank line, followed by imports from the local package.

The tcell label is also not needed, and should be removed, because the package is automatically imported using it's name tcell.  This is idiomatic for go when using versioned packages -- the v2 suffix is not the package name.